gtsfm/frontend/matcher/superglue_matcher.py

```diff
@@ -38,15 +38,16 @@ def __init__(self, use_cuda: bool = True, use_outdoor_model: bool = True):
             "sinkhorn_iterations": DEFAULT_NUM_SINKHORN_ITERATIONS,
         }
         self._use_cuda = use_cuda and torch.cuda.is_available()
+        self._model = SuperGlue(self._config).eval()
```

Comment:
are you planning on enabling cuda later? if not should this change remove all the use_cuda options above?

Reply:
Enabled it now.